src/test/java/api/support/fixtures/CirculationRulesFixture.java

```diff
@@ -66,12 +69,22 @@ public void updateCirculationRules(String rules) {
       .put("rulesAsText", rules);
 
     final Response response = putRules(circulationRulesRequest.encodePrettily());
-
     assertThat(String.format(
       "Failed to set circulation rules: %s", response.getBody()),
       response.getStatusCode(), is(204));
   }
 
+  public void updateCirculationRulesWithNonExistentPolicy(
+    String rules, String policyType) {
+
+    JsonObject circulationRulesRequest = new JsonObject()
+      .put("rulesAsText", rules);
+
+    final Response response = putRules(circulationRulesRequest.encodePrettily());
+    assertThat(String.format("The policy %s does not exist", policyType),
+      response.getStatusCode(), is(422));
+  }
```

Comment:
Why is this defined in the fixture, rather than an `attempt` method and matchers in the specific tests?

Apologies if my previous comment was unclear and led to this change.